### PR TITLE
Re-introduce 00 prefix

### PR DESCRIPTION
**Pull request by piotrpMSFT**
Also bumps the version suffix prefix [hehe] to preview1

/cc @livarcocc @jgoshi @eerhardt

### CODE REVIEW

**Comment by eerhardt** [comment]
Does this fix the RHEL builds?

**Comment by piotrpMSFT** [comment]
@eerhardt no. Working on RHEL now.

File: build/GitCommitInfo.targets [hunk 1]
```diff
@@ -11,8 +11,10 @@
     </Exec>
 
     <PropertyGroup>
+      <GitInfoCommitCount>$(GitInfoCommitCount.PadLeft(6,'0'))</GitInfoCommitCount>
+      
       <ShouldOverWriteThePropsFile 
-        Condition=" '$(CommitCount)' != '$(GitInfoCommitCount)' Or 
+        Condition=" 'ForceStringComparison$(CommitCount)' != 'ForceStringComparison$(GitInfoCommitCount)' Or
```
Comment (eerhardt) [inline build/GitCommitInfo.targets]: Why do we need this?

**Comment by piotrpMSFT** [comment]
@dotnet/dotnet-cli

**Pull request merged**